client/consul.go

```diff
@@ -21,7 +21,7 @@ const (
 	consulSubsystemName       = "consul"
 )
 
-//go:generate mockery --name=ConsulClientInterface --filename=consul_client.go --output=../mocks/client --tags=enterprise
```

Comment:
I thought I fixed this already but I guess I didn't, this is just to fix the filename for the consul client mock